BrickSelect/README.md

```diff
@@ -0,0 +1,17 @@
+# BrickSelect
+
+BrickSelect is a web component from PROGRAP developers who wanted an reusable and extendable drop-in replacement selector element.
+
+# #Installation
+`$ npm install @prograp/brick-slider`
+
+## Usage
+#### Element
+```
+<brick-select name="quantity" required>
+  <option value="">0,00</option>
+</brick-select>
+```
+#### Attributes
++ `auto-submit` ***automatic submitting the selected value***
```

Comment:
Description: "automatically submit the surrounding form once a value has been selected"